packages/nimble-components/src/text-area/index.ts

```diff
@@ -1,10 +1,11 @@
-import { attr } from '@microsoft/fast-element';
+import { attr, DOM, observable } from '@microsoft/fast-element';
```

Comment:
As I toggle error-visible on and off in storybook I see the icon jump around:

![iconjump](https://user-images.githubusercontent.com/1588923/229226308-59a1a8e1-aeaa-4db2-9f66-6df74c0ce20f.gif)

Guessing the issue is that the icon is shown before the initial scrollbar calculation happens. Guess you can have a class or something to only show the icon after calculation happens:

```js
// class prop
scrollbarWidth = -1;

// template
`<icon class="error-icon ${x => x.scrollbarWidth > 0 ? 'scrollbar-width-calculated' : '' }">`
```

or something

Reply:
Done